readme.md

```diff
@@ -516,6 +516,7 @@
 - [FFmpeg](https://github.com/transitive-bullshit/awesome-ffmpeg#readme) - Cross-platform solution to record, convert and stream audio and video.
 - [Icons](https://github.com/notlmn/awesome-icons#readme) - Downloadable SVG/PNG/font icon projects.
 - [Audiovisual](https://github.com/stingalleman/awesome-audiovisual#readme) - Lighting, audio and video in professional environments.
+- [VLC/LibVLC](https://github.com/mfkl/awesome-vlc#readme) - The ultimate media player app and SDK.
```

Comment:
```suggestion
- [VLC](https://github.com/mfkl/awesome-vlc#readme) - The ultimate media player app and SDK.
```

And the description should describe VLC and not be a marketing tagline.

Reply:
Done